L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -968,6 +975,28 @@ namespace trklet {
 
     double stripLength_PS_{0.1467};
     double stripLength_2S_{5.0250};
+
+    //Following values are used for duplicate removal
+    //Variable bin edges for 12 bins.
+    std::vector<double> varrinvbins_{{-rinvcut(),
+                                      -0.005433,
+                                      -0.004968,
+                                      -0.004459,
+                                      -0.003828,
+                                      -0.002911,
+                                      0,
+                                      0.002911,
+                                      0.003828,
+                                      0.004459,
+                                      0.004968,
+                                      0.005433,
+                                      rinvcut()}};
```

Comment:
I think this should be defined with single curly brackets -- why does it have 2?

Reply:
Oh I see you're dereferencing in the getter -- I think this definitely shouldn't be like this and is probably unsafe from a memory perspective. Try doing this with single curly brackets and remove the & in the getter.

Reply:
This is a holdover from when I had multiple vectors of bin edges listed. I just never deleted the second set of brackets. I'll go ahead and delete.